angularjs-portal-home/src/main/resources/endpoint.properties.example

```diff
@@ -1,51 +0,0 @@
-#
-# Licensed to Jasig under one or more contributor license
-# agreements. See the NOTICE file distributed with this work
-# for additional information regarding copyright ownership.
-# Jasig licenses this file to you under the Apache License,
-# Version 2.0 (the "License"); you may not use this file
-# except in compliance with the License. You may obtain a
-# copy of the License at:
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on
-# an "AS IS" BASIS,WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND,either express or implied. See the License for the
-# specific language governing permissions and limitations
-# under the License.
-#
-
-#key.uri
-#key.username
-#key.password
-#key.attributes (csv)
-
-#Wiscmail lookup
-wiscmaillookup.uri=https://wisctest.wisc.edu/myuw.json?action=login_info&account={uid}
-wiscmaillookup.username=
-wiscmaillookup.password=
-wiscmaillookup.attributes=uid
-
-#new stuff feed
-newstuff.uri=http://localhost:8080/web/staticFeeds/new-stuff.json
-newstuff.username=
-newstuff.password=
-newstuff.attributes=uid
-
-#notifications feed
-notifications.uri=http://localhost:8080/web/staticFeeds/sample_notification.json
-notifications.username=
-notifications.password=
-notifications.attributes=uid
-
-#subreddit feed example
-uwmadisonreddit.uri=https://www.reddit.com/r/uwmadison/hot.json
-uwmadisonreddit.username=
-uwmadisonreddit.password=
-uwmadisonreddit.attributes=
-
-wiscdirectory.uri=http://www.wisc.edu/directories/json
-
-wiscedusearch.uri=http://ajax.googleapis.com/ajax/services/search/web
```

Comment:
Easy to do. I found doing a `git status` after a `git add --all` can help identify when doing things.